napari/layers/shapes/_tests/test_shapes.py

```diff
@@ -1891,7 +1891,7 @@ def test_copy_and_paste():
     layer.selected_data = {0, 1}
     layer._copy_data()
     layer._paste_data()
-    assert len(layer._clipboard) == 6
+    assert len(layer._clipboard) > 0
```

Comment:
The reason I changed this is because whenever we add a new type of thing to the clipboard (in the larger #3327, it was text color), the exact number of keys changes. But it seems like the test should really just care that there is something in the clipboard (not exactly how many things there are).